test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -17,19 +17,24 @@
 
 import java.io.BufferedReader;
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.util.Map;
-
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
 import com.google.common.io.Files;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.apache.cassandra.bridges.Bridge;
 import org.apache.cassandra.htest.Config;
+import org.apache.commons.io.FileUtils;
```

Comment:
These imports aren't grouped correctly, but I just have my IDE handle that after I merge.